cfgov/data_research/scripts/export_public_csvs.py

```diff
@@ -117,7 +117,7 @@ def export_downloadable_csv(geo_type, late_value):
     Each CSV is to start with a National row for comparison.
 
     CSVs are posted at
-    http://files.consumerfinance.gov.s3.amazonaws.com/data/mortgage-performance/downloads/  # noqa: E501
+    https://files.consumerfinance.gov.s3.amazonaws.com/data/mortgage-performance/downloads/  # noqa: E501
```

Comment:
See above; remove s3.amazonaws.com.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3998/commits/6f010183f80d223c28f5d33d179b59cf995d9d86